lib/ui/platform_dispatcher.dart

```diff
@@ -347,7 +356,13 @@ class PlatformDispatcher {
   // ignore: unused_field, field will be used when platforms other than web use these focus APIs.
   Zone _onViewFocusChangeZone = Zone.root;
   set onViewFocusChange(ViewFocusChangeCallback? callback) {
-    _onViewFocusChange = callback;
+    _onViewFocusChange = (ViewFocusEvent event) {
+      _focusedViewId = switch (event.state) {
```

Comment:
Does this mean that `PlatfromDispatcher.focusedViewId` is only updated when someone has set a onViewFocusChange callback on the PlatformDispatcher? That seems like odd/confusing behavior.